src/to-contain-html.js

```diff
@@ -1,10 +1,20 @@
 import {checkHtmlElement} from './utils'
 
+function getNormalizedHtml(container, htmlText) {
+  const div = container.ownerDocument.createElement('div')
+  div.innerHTML = htmlText
+  return div.innerHTML
+}
+
 export function toContainHTML(container, htmlText) {
   checkHtmlElement(container, toContainHTML, this)
 
+  if (typeof htmlText !== 'string') {
+    throw new Error(`.toContainHTML() expects a string value, got ${htmlText}`)
+  }
```

Comment:
Without this change, the `nonExistantElement` tests started failing. This value is basically an equivalent of `expect(element).toContainHTML(null)` which turned into a `string.includes(null)`, producing `false`. 

Now the `htmlText` always gets converted to a string, so this extra check is needed to keep the assertions failing where they were before.